CI failure in unit tests:

```
>           self.assertEqual(e.message, 'Decryption failed (no vault secrets would found that could decrypt)')
E           AssertionError: 'Decryption failed (no vault secrets were found that could decrypt)' != 'Decryption failed (no vault secrets would found that could decrypt)'
E           - Decryption failed (no vault secrets were found that could decrypt)
E           ?                                      ^^^
E           + Decryption failed (no vault secrets would found that could decrypt)
E           ?                                      ^^^^

test/units/parsing/yaml/test_loader.py:204: AssertionError
```

--- a/lib/ansible/parsing/vault/__init__.py
+++ b/lib/ansible/parsing/vault/__init__.py
@@ -587,7 +587,7 @@ class VaultLib:
                              (vault_secret_id, filename, e))
                 continue
         else:
-            msg = "Decryption failed (no vault secrets would found that could decrypt)"
+            msg = "Decryption failed (no vault secrets were found that could decrypt)"
             if filename:
                 msg += " on %s" % filename
             raise AnsibleVaultError(msg)

--- a/test/units/parsing/yaml/test_loader.py
+++ b/test/units/parsing/yaml/test_loader.py
@@ -201,7 +201,7 @@ class TestAnsibleLoaderVault(unittest.TestCase, YamlTestUtils):
             self.vault.decrypt(ciphertext)
         except Exception as e:
             self.assertIsInstance(e, errors.AnsibleError)
-            self.assertEqual(e.message, 'Decryption failed (no vault secrets would found that could decrypt)')
+            self.assertEqual(e.message, 'Decryption failed (no vault secrets were found that could decrypt)')
 
     def _encrypt_plaintext(self, plaintext):
         # Construct a yaml repr of a vault by hand

